field.h

```diff
@@ -0,0 +1,36 @@
+#ifndef FIELD_H
+#define FIELD_H
+#include "cell.h"
+#include <vector>
+#include <list>
+
+class Field{
+public:
+    //Field();
+    Field(int fieldSize = 5);
+    int getSize(){return size;}
+
+    std::vector<std::vector<Cell>>& getField(){return field;}
+    bool getMovableStatus(){return isMovable;}
+
+    void updateMovability(); //Update field movability
+    int moveCells(Direction dir); //Move all filled cells on field to direction dir
+    void cleanField(); //Set all cells value as 0
+    void createField(int fieldSize = 5); //Create field with new size
+    void printFieldConsole(); //Print field to console (this function just for cheking)
+
+    void FillFreeCells(int numCellsToFill = 1);//Add random numbers to random cells on field
```

Comment:
numCellsToFill is too much of the same details in the name, toFill is not needed here, Cells is questionable

Reply:
Also remove default parameter here we don't need it

Reply:
done